scripts/startNewSchoolYear.js

```diff
@@ -0,0 +1,55 @@
+const appPromise = require('../src/app');
+
+const { info, error } = require('../src/logger');
+
+const { yearModel, schoolModel } = require('../src/services/school/model');
+const federalStateModel = require('../src/services/federalState/model');
+
+const exceptFederalStateNames = ['Brandenburg'];
+
+const NEXT_SCHOOL_YEAR = '2021/22';
+const MAINTENANCE_START_DATE = new Date('2021-08-01');
+
+appPromise
+	.then(async () => {
+		const nextSchoolYearId = await yearModel.findOne({ name: NEXT_SCHOOL_YEAR }).select('_id').lean().exec();
```

Comment:
why not exporting the script function which takes NEXT_SCHOOL_YEAR and MAINTAINANCE_START_DATE here without require the appPromise? Then the script itself could be moved into a business component part of the application.

Reply:
Will be done in the follow-up-ticket to end the transfer period.